lib/filewriter.js

```diff
@@ -139,13 +139,13 @@ FileWriter.prototype = {
             function finish(err) {
                 if (guard && global.clearTimeout) clearTimeout(guard);
                 guard = null;
-                if (fd !== null) fs.close(fd, function(){});
+                if (fd !== null) try { fs.closeSync(fd) } catch (e) { }
```

Comment:
it doesn't affect correct operation, but we can return the error at least, sure, to let the caller know
that there was a problem.  A close error could in theory leak a file descriptor (the possible errors from
the close system call are EBADF, EINTR and EIO)